shared/api/cluster.go

```diff
@@ -60,4 +60,6 @@ type ClusterMember struct {
 
 	// API extension: clustering_roles
 	Roles []string `json:"roles" yaml:"roles"`
+	// API extension: multi_architecture_clustering
+	Arch string `json:"arch" yaml:"arch"`
```

Comment:
Need a line break before that new block, also, call it `Architecture` instead of `Arch` and set the json/yaml names to `architecture`.